### PR TITLE
update owners

### PR DESCRIPTION
**Pull request by danmosemsft**

### CODE REVIEW

**Comment by danmosemsft** [comment]
@karelz please review.

File: Documentation/project-docs/issue-guide.md [hunk 1]
```diff
@@ -48,49 +48,49 @@ Areas are tracked by labels area-&#42; (e.g. area-System.Collections). Each area
...
 | [System.Buffers](https://github.com/dotnet/corefx/labels/area-System.Buffers)                 | [@safern](https://github.com/safern) |  |
-| [System.CodeDom](https://github.com/dotnet/corefx/labels/area-System.CodeDom)                 | [@Priya91](https://github.com/Priya91) |  |
-| [System.Collections](https://github.com/dotnet/corefx/labels/area-System.Collections)         | [@ianhays](https://github.com/ianhays), [@safern](https://github.com/safern) | </ul>Excluded:<ul><li>System.Array -> System.Runtime</li></ul> |
-| [System.ComponentModel](https://github.com/dotnet/corefx/labels/area-System.ComponentModel)   |  [@safern](https://github.com/safern), [@AlexGhiondea](https://github.com/AlexGhiondea) |  |
+| [System.CodeDom](https://github.com/dotnet/corefx/labels/area-System.CodeDom)                 | [@pjanotti](https://github.com/pjanotti) |  |
+| [System.Collections](https://github.com/dotnet/corefx/labels/area-System.Collections)         |**[@safern](https://github.com/safern)**,  [@ianhays](https://github.com/ianhays) | </ul>Excluded:<ul><li>System.Array -> System.Runtime</li></ul> |
```
Comment (nchikanov) [inline Documentation/project-docs/issue-guide.md]: What's the point of the "**"s here?

Comment (danmosemsft) [inline Documentation/project-docs/issue-guide.md]: bolding the primary owner :smiley:

**Comment by danmosemsft** [comment]
@karelz good to go?

**Pull request merged**